packages/now-next/src/index.ts

```diff
@@ -222,9 +222,6 @@ export const build = async ({
 
     // If this is the initial build, we want to start the server
     if (!urls[entrypoint]) {
-      debug(`${name} Installing dependencies...`);
```

Comment:
This one is already within a `meta.isDev` conditional, so just remove it completely.